cmd/plural/cd.go

```diff
@@ -117,6 +117,15 @@ func (p *Plural) doInstallOperator(url, token, values string) error {
 	if err != nil {
 		return err
 	}
+	alreadyExists, err := console.IsAlreadyAgentInstalled(p.Kube.GetClient())
```

Comment:
we potentially should add an argument here to not do this check on `plural cd clusters reinstall` (the expectation is you are updating an already existing agent)